src/content/special-interest-groups/meta-omics/index.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Introduction
+subtitle: 'Community of researchers investigating biological community composition and dynamics using metagenomic, metatranscriptomic, and metaproteomic approaches.'
+groupName: Meta-omics
+leads:
+  - jfy133: James Fellows Yates
+  - erikrikarddaniel: Daniel Lundin
+  - carsonjm: Carson J. Miller
+pipelines:
+  - ampliseq
+  - createtaxdb
+  - differentialabundance
+  - eager
+  - funcscan
+  - mag
+  - magmap
+  - metatdenovo
+  - phageannotator
+  - phyloplace
+  - rnaseq
+  - taxprofiler
+---
+
+# Introduction
+
+Meta-omics methods aim to simultaneously analyse the entire multi-species potential and realised functional capacity of a biological sample. In other words to taxonomically and functionally characterise all organisms and their activity in a community.
+
+The initial aims of this interest group will be:
+
+1. helping coordinate compatibility between existing and potential meta-omic nf-core pipelines
+2. implementing semi-'chaining' of sequential pipelines
+3. developing standard parameters for different sub-disciplines (i.e. microbiome analysis) for each pipeline
+4. documenting usage of 'first level' meta-omic pipelines followed by complementary analyses with "secondary" pipelines such as #funcscan, #phageannotator, #phyloplace, #rnaseq, #differentialabundance etc.
+
+Depending on interest/activity, other potential activities could include:
+
+1. benchmarking meta-omic pipelines using relevant datasets (i.e. CAMI2), or reads generated using #readsimulator
+2. adding nf-tests to meta-omic pipelines
```

Comment:
A bit confused what  this is referring to now :sweat_smile:

Reply:
I leave this one to you two 😅 Otherwise LGTM to merge 👍🏻